src/app/(documentation)/docs/[...slug]/page.tsx

```diff
@@ -0,0 +1,30 @@
+import { DocumentationLayout } from '@/components/layout/documentation';
+import { ROOT_DOCS_DIRECTORY, getAllDocsMeta, getDocBySlug } from '@/lib/mdx/docs';
+import { findFile } from '@/lib/utils/find-file';
+import { getHeadingLinks } from '@/lib/utils/get-heading-links';
+
+export async function generateStaticParams() {
+  const docs = await getAllDocsMeta();
+
+  return docs.map(doc => ({
+    slug: doc.parent ? [doc.parent, doc.slug] : [doc.slug],
+  }));
+}
+
+type DocPageProps = {
+  params: { slug: string[] };
+};
+
+export default async function DocsPage({ params }: DocPageProps) {
+  const { slug } = params;
+  const realSlug = slug[slug.length - 1];
+  const { title, content: Content } = await getDocBySlug(realSlug);
+  const [filePath] = findFile(ROOT_DOCS_DIRECTORY, realSlug, '.mdx', true);
+  const links = await getHeadingLinks(filePath);
+
+  return (
+    <DocumentationLayout title={title} links={links}>
```

Comment:
Could you check the `DocumentationLayout` styles? It seems a border at the bottom is missing:
<img width="1009" alt="Screenshot 2024-03-22 at 14 19 53" src="https://github.com/bejamas/httptoolkit-website/assets/22168189/728ededb-7b59-4c6a-b9a0-3bed301aacce">